src/Vixen.Modules/App/ExportWizard/BulkExportSourcesStage.cs

```diff
@@ -150,7 +150,7 @@ private void btnDelete_Click(object sender, EventArgs e)
 
 		private void btnAddAll_Click(object sender, EventArgs e)
 		{
-			var files = Directory.GetFiles(SequenceService.SequenceDirectory, "*.tim", SearchOption.AllDirectories);
+			var files = Directory.GetFiles(SequenceService.SequenceDirectory, "*.tim", SearchOption.TopDirectoryOnly);

```

Comment:
It should just exclude the auto_backup folder. Other folders are ok to recurse into.